build.sh

```diff
@@ -0,0 +1,51 @@
+name="oval-graph"
+module="oval_graph"
+
+git checkout master
+
+# Edit version, change log, etc... 
+vim ${module}/version.py
+vim ~/rpmbuild/SPECS/${name}.spec
+
+version=$(python3 setup.py --version)
+
+# Version commit
+git add -A
+git commit -m "${version}"
+git tag "${version}"
+git push upstream master
+
+# Build for Pipy
+rm -r dist
+python3 setup.py pytest sdist bdist_wheel
+twine upload dist/*
+
```

Comment:
I would end the release script here, and add a build script for Fedora to some subdirectory, as it is not directly related to the project.